projects/L4T/packages/xorg-server/package.mk

```diff
@@ -0,0 +1,152 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="xorg-server"
+PKG_VERSION="1.20.10"
+#PKG_SHA256="a81d8243f37e75a03d4f8c55f96d0bc25802be6ec45c3bfa5cb614c6d01bac9d"
+PKG_LICENSE="OSS"
+PKG_SITE="http://www.X.org"
+PKG_URL="http://xorg.freedesktop.org/archive/individual/xserver/$PKG_NAME-$PKG_VERSION.tar.bz2"
+PKG_DEPENDS_TARGET="toolchain util-macros font-util xorgproto libpciaccess libX11 libXfont2 libXinerama libxshmfence libxkbfile libdrm openssl freetype pixman systemd xorg-launch-helper"
+PKG_NEED_UNPACK="$(get_pkg_directory xf86-video-nvidia) $(get_pkg_directory xf86-video-nvidia-legacy)"
+PKG_LONGDESC="Xorg is a full featured X server running on Intel x86 hardware."
+PKG_TOOLCHAIN="autotools"
+
+get_graphicdrivers
+
+if [ "$COMPOSITE_SUPPORT" = "yes" ]; then
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libXcomposite"
+  XORG_COMPOSITE="--enable-composite"
+else
+  XORG_COMPOSITE="--disable-composite"
+fi
+
+if [ ! "$OPENGL" = "no" ]; then
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET $OPENGL libepoxy"
+  if [ ! "$PROJECT" = "L4T" ]; then 
+    XORG_MESA="--enable-glx --enable-dri --enable-glamor"
+  else  
+    XORG_MESA="--enable-glx --enable-dri --disable-glamor"
+  fi
```

Comment:
since the package in under the project L4T, are these tests necessary?
adding the desired options directly to the configure options should be OK.

Reply:
I just copied the newer version from upstream libreelec. and added some extra patches to fix building against libglvnd. I plan to clean up the whole tree, once I rework the BSP/Kernel stuff. Have plans to make it more configurable and easier for people porting other L4T platforms to lakka. When that is done, we will probably have a lot of changes in the L4T specific packages.